index.html

```diff
@@ -0,0 +1,238 @@
+    <link rel="stylesheet" href="styles.css" type="text/css">
+    <title>WebTekhy | Homepage</title>
+</head>
+<body id="hideOverflow">
+    <header class="header" id="header">
+        <div class="logo">
+            <a href="index.html"><img class="logo-img" src="images/webtekhy-icon.png" alt="webtekhy icon"></a>
+        </div>
+        <nav id="menu_bar" class="navbars">
+            <i id="menuBarImg" class="fa fa-bars" aria-hidden="true"></i>
+            <div id="close_bar" class="close-menu">
+                <i id="close_menu" class="close" >&times;</i>  
+            </div>
+            <ul id="menu_list" class="navis">
+                <li class="nav-item"><a href="about.html" class="nav-link abt-nav">About</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Sponsor</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+                <li class="nav-item"><a href="#" class="nav-link wt-nav">WT Campaign</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="title-section">
+            <div class="details">
+                <p class="quote-text">
+                "Hello! Sharing World"
+                </p>
+                <h1 class="title">
+                    WEBTEKHY UNIVERSAL BOOTCAMP 2022
+                </h1>
+            </div>
+            
+            <div class="descriptions">
+                <p class="description">
+                    A fun festival where people from more than 80 countries who believe in the value of sharing and openness and 
+                    create positive change gather and share their thoughts will be held in Korea in October.
+                </p>
+            </div>
+
+            <h2 class="date">
+                2022.05.16(MON) ~ 20(FRI)
+            </h2>
+            <p class="location">
+                @ National Theartre of Nigeria, Kwara Hotel and more
+            </p>
+        </section>
+
+        <section class="program-section">
+            <h2 class="program-title">Our Program</h2>
+            <div class="bottom-line"></div>
+            <div class="programs">
+                <div class="prog program-1">
+                    <div class="program-icon-img transparency">
+                        <img src="images/program_icon_01.png" alt="Lecture Icon">
+                    </div>
+                    <h3 class="prog-title title-1">Lecture</h3>
+                    <div class="about-prog">
+                        <p class="prog-details transparency lecture-details">
+                            Listen to lectures from speakers from around the world and 
+                            learn about the latest trends in the world.
+                        </p>
+                    </div>
+                </div>
```

Comment:
- [ ] Kindly replace the `div` tags with the semantic `article` tags in both your program and speaker cards.